packages/x-charts/src/BarChart/useBarPlotData.ts

```diff
@@ -170,3 +178,26 @@ function getBandSize({
     offset,
   };
 }
+
+function getValueCoordinate(
+  isVertical: boolean,
+  minValueCoord: number,
+  maxValueCoord: number,
+  baseValue: number,
+  minBarSize: number,
+): { barSize: number; startCoordinate: number } {
+  const isSizeLessThanMin = Math.abs(maxValueCoord - minValueCoord) < minBarSize;
+  const barSize = isSizeLessThanMin ? minBarSize : Math.abs(maxValueCoord - minValueCoord);
```

Comment:
Before we didn't need `Math.abs`, why do we need it now? Isn't `maxValueCoord` always greater than `minValueCoord`, so the result is always non-negative?

Reply:
Not needed yeah, we already run math.min/math.max to get the coords